solr/core/src/java/org/apache/solr/search/SignificantTermsQParserPlugin.java

```diff
@@ -28,9 +28,7 @@
 import org.apache.lucene.index.PostingsEnum;
 import org.apache.lucene.index.Terms;
 import org.apache.lucene.index.TermsEnum;
-import org.apache.lucene.search.DocIdSetIterator;
-import org.apache.lucene.search.IndexSearcher;
-import org.apache.lucene.search.Query;
+import org.apache.lucene.search.*;
```

Comment:
please don't use wildcard imports.

Reply:
Done. 
I've found many wildcard imports in the Solr codebase. Don't you think it's worth to add an editorconfig to enforce that rule?